speckle/resources/objects.py

```diff
@@ -4,24 +4,35 @@
 from pydantic import BaseModel, validator
 from typing import List, Optional
 from speckle.base.resource import ResourceBase, ResourceBaseSchema
-from speckle import objects
-
 
 NAME = 'objects'
 METHODS = ['list', 'get', 'update', 'create'
            'delete', 'comment_get', 'comment_create',
            'get_bulk', 'set_properties']
 
 
-class SpeckleObject:
+class SpeckleObject(ResourceBaseSchema):
```

Comment:
What's going on here? How come this counts as a change when the master branch already has this? Would you mind refreshing your changes from the upstream master branch and recommitting so these non-changes don't appear here?

Reply:
Yeps, sorry, I've merged the latest changes from upstream master.

Reply:
Yeah that works, commented on the PR you made but it's just me being picky: https://github.com/speckleworks/PySpeckle/pull/20#issuecomment-518197590